packages/paste-core/components/timeline/src/TimelineItemGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, type BoxProps } from "@twilio-paste/box";
+import { DetailText } from "@twilio-paste/detail-text";
+import * as React from "react";
+
+import { TimelineContext, TimelineGroupContext } from "./TimelineContext";
+import { TimelineItemIcon } from "./TimelineItemIcon";
+import type { TimelineItemProps } from "./types";
+
+export const TimelineItemGroup = React.forwardRef<
```

Comment:
suggestion(blocking): create TimelineItemGroupProps so that we can export it from the package. we like to export the types of each component we export as part of the API. that's also how the API docs are created

also use `HTMLPasteProps("li")` and `HTMLLIElement` for any components that are rendered as `li`s

optional: move to types.ts file